srcpkgs/betterlockscreen/template

```diff
@@ -0,0 +1,20 @@
+# Template file for 'betterlockscreen'
+pkgname=betterlockscreen
+version=3.0.1
+revision=1
+archs=noarch
+depends="i3lock-color ImageMagick bc xrandr feh xdpyinfo"
```

Comment:
Should probably depend on bash, too. (bash script and none of the above dependencies pulls in bash. Bash might not be installed by default).

Reply:
Whoa, good point. Missed that one!